utils/provided-goi-analysis-functions.R

```diff
@@ -184,21 +183,57 @@ plot_goi_expression_sina <- function(normalized_sce,
         )
       )
     
+    # calculate average gene expression
+    avg_gene_exp=colMeans(logcounts(normalized_sce))
```

Comment:
I think the reason you see a thick line is because this is actually a vector of all the average means. So you have a lot of values you are giving to the `yintercept` option in the form of a vector. We need to actually take the mean of the means here. 
```suggestion
    avg_gene_exp=mean(colMeans(logcounts(normalized_sce)))
```